src/app/[room]/page.tsx

```diff
@@ -0,0 +1,48 @@
+'use client'
+
+import { useEffect, useState } from 'react'
+import { socket } from '@/socket'
+import SocketIoTestSection from '@/components/socketIoTest/SocketIoTestSection'
+import StoryPoints from '@/components/StoryPoints'
+import CardsContainer from '@/components/CardsContainer'
+
+export type ChatEvent = {
+	message: string
+	timeStamp: string
+	room: string
+}
+
+export default function UserRooms({ params }: { params: { room: string } }) {
+	const [roomId, setRoomId] = useState('')
+	const [userName, setUserName] = useState('')
+	// const [chatEvents, setChatEvents] = useState<ChatEvent[]>([])
+
+	// biome-ignore lint/correctness/useExhaustiveDependencies: <explanation>
+	useEffect(() => {
+		const { room } = params
+
+		console.log('%c>>> on user page load roomId:', 'color: red', room)
+
+		const user = 'Bob'
+
+		socket.emit('join-room', room, user)
+
+		setRoomId(room)
+		setUserName(user)
+	}, [])
+
+	return (
+		<main className='px-16 py-12 flex flex-col items-center gap-12 w-full'>
+			<h1 className='text-3xl text-green-400'>USER Scrum Diving Room</h1>
```

Comment:
todo: Prefer using CSS for all-caps words, it's [better for accessibility](https://www.timdunklee.com/notes/use-css-text-transform-for-uppercase-letters-a11y/)

Reply:
That's not the actual page title. It's just there for now so I know what page I'm on. I'm mostly focused on functionality and layout. This will 100% be gone. But I will keep that in mind for all caps!!